src/classes/UTIL_Namespace.cls

```diff
@@ -57,6 +57,15 @@ public class UTIL_Namespace {
         }
         return plainNamespace;   
     }
+    
+    /*******************************************************************************************************
+    * @description Returns class name without the namespace (if any)
+    */
+    public static String removeNamespace(String className) {
```

Comment:
@RadSFDeveloper Since this is limited to removing the namespace for dot notation formats only (effectively just for Apex Classes), should this be renamed to `removeClassNamespace` or `removeDotNotationNamespace`? Alternatively, could the existing (and poorly named) `StrTokenRemoveNSPrefix` method support removing the namespace prefix for both `__` and `.` notation formats?

Reply:
I think separation of removing namespace `__` (sObjects) and `.` (classes) is better since it affects less code so testing of this PR is restricted on exactly the change required by the PR.
I can rename the method to be more descriptive (even though the argument indicates it is the class name) and reuse the `getNamespace()`.

Reply:
Actually, using `getNamespace()` does not support testable code (existing remove class namespace unit test failed), so I have just renamed the method to the `removeClassNamespace()`.